mod/gc_group_layout/lib/groups.php

```diff
@@ -354,8 +354,12 @@ function groups_handle_profile_page($guid) {
 	groups_register_profile_buttons($group);
 
 	$content = elgg_view('groups/profile/layout', array('entity' => $group));
-	$sidebar = elgg_view('groups/sidebar/sidebar', array('entity' => $group));
-    
+	$sidebar_value = $group->getPrivateSetting('group_tools:cleanup:menu');
+
+	if($sidebar_value == 'yes'){
```

Comment:
This will need to test for 'no'. The form says "hide sidebar items", when I select 'Yes' it displays the items, and when I select 'No' it hides them.

Reply:
Oups, you are right, I made the change!